magda-registry-api/src/test/scala/au/csiro/data61/magda/registry/ApiSpec.scala

```diff
@@ -47,6 +48,26 @@ abstract class ApiSpec extends FunSpec with ScalatestRouteTest with Matchers wit
 
   val databaseUrl = Option(System.getenv("POSTGRES_URL")).getOrElse("jdbc:postgresql://localhost:5432/postgres")
 
+  def addTenantIdHeader(tenantId: BigInt): RawHeader = {
+    RawHeader(MAGDA_TENANT_ID_HEADER, tenantId.toString)
+  }
+
+  val SINGLE_TENANT_ID = MAGDA_ADMIN_PORTAL_ID
+
+  def addSingleTenantIdHeader: RawHeader = {
+    addTenantIdHeader(SINGLE_TENANT_ID)
+  }
+
+  def addAdminPortalIdHeader: RawHeader = {
+    addTenantIdHeader(MAGDA_ADMIN_PORTAL_ID)
+  }
+
+  // Any positive numbers
+  val tenant_1: BigInt = 1
```

Comment:
Can these either be in camelCase or UPPER_CASE? [The scala conventions](https://docs.scala-lang.org/style/naming-conventions.html) say that constants should be UpperCamelCase, but I'm pretty sure we've used UPPER_CASE for constants in other parts so that'd be fine too.

Reply:
done